mvrx/src/test/kotlin/com/airbnb/mvrx/PersistedStateTest.kt

```diff
@@ -238,6 +238,58 @@ class PersistedStateTest : BaseTest() {
         persistMavericksState(State())
     }
 
+    @Test
+    fun testClassWithExactly32Parameters() {
+        data class StateWith32Params(
+            val p0: Int = 0,
```

Comment:
does it matter that there are only 16 persist state annotations here? it's confusing why only half of the params have it, but I suppose it shouldn't matter for the purposes of this bug?

Reply:
It's the number of parameters in the constructor that trigger the bug not the number of persisted elements, I copied this from another test, that is why this is like this, but we could also persist them all or persist none of them.
The test was failing before the fix.